elasticdl/python/ps/embedding_table.py

```diff
@@ -57,6 +64,27 @@ def set(self, indices, values):
     def clear(self):
         self.embedding_vectors.clear()
 
+    def to_tensor(self):
+        """Convert the embedding table to elasticDL Tensor"""
+        indices = []
+        embedding_vectors = []
+        for id, embedding_vector in self.embedding_vectors.items():
+            indices.append(id)
+            embedding_vectors.append(embedding_vector)
+        return Tensor(
+            values=np.array(embedding_vectors),
+            indices=np.array(indices),
+            name=self.name,
+        )
+
+    def to_embedding_table_info_pb(self):
+        """Convert the embedding table information to a protobuf"""
+        embedding_pb = EmbeddingTableInfo()
+        embedding_pb.name = self.name
+        embedding_pb.dim = self.dim
+        embedding_pb.initializer = str(self.initializer_value)
```

Comment:
How do we differentiate whether the initializer_value is float or str while restoring from pb?

Reply:
In `__init__` of EmbeddingTable, we differentiate by `is_slot`.
https://github.com/sql-machine-learning/elasticdl/blob/b3fd10197d0f1184838e4e533865ca162570ccef/elasticdl/python/ps/embedding_table.py#L35-L42